test/safe_pg_migrations_test.rb

```diff
@@ -193,6 +193,26 @@ def up
     end
   end
 
+  def test_create_table_idem_potent
+    @connection.create_table(:users) { |t| t.string :email }
+    @migration =
+      Class.new(ActiveRecord::Migration::Current) do
+        def change
+          create_table :users do |t|
+            t.string :email
+          end
+        end
+      end.new
+
+    write_calls = record_calls(@migration, :write) { run_migration }.map(&:first)
+
+    assert_equal [
+      '== 8128 : migrating ===========================================================',
+      '-- create_table(:users)',
+      "   -> /!\\ Table 'users' already exists. Skipping statement.",
```

Comment:
```suggestion
      "   -> /!\ Table 'users' already exists. Skipping statement.",
```

Reply:
![image](https://user-images.githubusercontent.com/9265111/94277519-f832dd80-ff49-11ea-87ed-c20eb110184b.png)

It looks like backslash needs to be escaped. Am I missing something?

Reply:
Should I use single quote instead? And escape the quotes around `users`?

Reply:
don't know ^^ not blocking anyway

Reply:
OK, I'm keeping the `"\\"` as it's used in the rest of the code